doc/site/layouts/docs/lua_api.html

```diff
@@ -1,16 +1,325 @@
+{{/*
+	I will be real with you, this is completely unreadable.
+	This is in several parts:
+	1. Analyze and generate a table of contents
+	2. Generate API cards
+	3. Type backlinks
+
+	The API generator is recursive, and generates the style and heading based on the level.
+*/}}
+
 {{ define "main" }}
 	<div class='hx-mx-auto hx-flex {{ partial "utils/page-width" . }}'>
-	{{ partial "entries_sidebar.html" (dict "context" .) }}
```

Comment:
You can remove [this component](https://github.com/beyond-all-reason/RecoilEngine/blob/88207e2/doc/site/layouts/partials/entries_sidebar.html)